pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -187,48 +208,45 @@ func (s *Service) HandleOutbound(msg *service.DIDCommMsg, destination *service.D
 }
 
 func (s *Service) handle(msg *message) error {
-	logger.Infof("entered into private handle didcomm message: %s ", msg)
+	logger.Infof("entered into private handle didcomm message: %s ", msg.ThreadID)
 
 	next, err := stateFromName(msg.NextStateName)
 	if err != nil {
 		return fmt.Errorf("invalid state name: %w", err)
 	}
 	logger.Infof("next valid state to transition -> %s ", next.Name())
-
 	for !isNoOp(next) {
-		// TODO change from thread id to connection id #397
-		// TODO pass invitation id #397
 		s.sendMsgEvents(&service.StateMsg{
 			Type: service.PreState, Msg: msg.Msg, StateID: next.Name(),
-			Properties: createEventProperties(msg.ThreadID, "")})
+			Properties: createEventProperties("", "")})
```

Comment:
We need to reconsider where we are creating the connectionID. Without any correlation, the consumers will not be able to fetch connection details (if required). For now, can we add a TODO ?

Reply:
Added TODO